src/DIRAC/Resources/LogFilters/test/Test_LogFilter.py

```diff
@@ -66,3 +51,51 @@ def mf():
 )
 def test_mf(mf, name, level, result):
     assert mf.filter(Record("blabla", name=name, level=level)) == result
+
+
+@pytest.fixture
+def pf():
+    options = {"Accept": "some,Words", "Reject": "Foo"}
+    pf = PatternFilter(options)
+    assert pf._accept == ["some", "Words"]
+    assert pf._reject == ["Foo"]
+    return pf
+
+
+@pytest.mark.parametrize(
+    "record, result",
+    [
+        (("Some",), False),
+        (("some, Words",), True),
+        (("some, Words", "Foo"), False),
+    ],
+)
+def test_pf(pf, record, result):
+    assert pf.filter(Record(*record)) == result
+
+
+@pytest.fixture
+def sdf():
+    sdf = SensitiveDataFilter()
+    return sdf
+
+
+@pytest.mark.parametrize(
+    "record, result",
+    [
+        (("Message", "Variable message"), ("Message", "Variable message")),  # nothing to filter
+        (
+            ("Message", "blablabla -----BEGIN CERTIFICATE-----\n12345\n45678\n-----END CERTIFICATE----- blablabla"),
+            ("Message", "blablabla ***REDACTED*** blablabla"),
+        ),  # should not display the certificate
+        (
+            ("-----BEGIN CERTIFICATE-----\n12345\n45678\n-----END CERTIFICATE-----", "Variable message"),
+            ("-----BEGIN CERTIFICATE-----\n12345\n45678\n-----END CERTIFICATE-----", "Variable message"),
+        ),  # should display the certificate because it is in the fixed part (assuming that it is displayed on purpose in this case)
+    ],
+)
+def test_sdf(sdf, record, result):
+    recordInstance = Record(*record)
+    sdf.filter(recordInstance)
```

Comment:
Is there a reason for using a fixture here rather than:

```suggestion
    SensitiveDataFilter().filter(recordInstance)
```